source/manuals/programming-languages/nodejs/index.html.md.erb

```diff
@@ -82,24 +120,343 @@ Keeping tests in the same directory makes them easier to find and more obvious w
 |   └── product.njk
 ```
 
-#### Use a config directory
-Keep your configuration together in a config folder.
 
+See also:
+
+- [Node.js Project Structure Tutorial](https://blog.risingstack.com/node-hero-node-js-project-structure-tutorial/) on [Rising Stack](https://blog.risingstack.com/)
+
+## Language constructs
+
+### Declarations
+
+> Use `const` and `let`, avoid `var`
+
+Embrace immutability (more below) and don't let
+[hoisting](http://www.adequatelygood.com/JavaScript-Scoping-and-Hoisting.html#declarations_names_and_hoisting)
+confuse you.
+
+
+### Functions
+
+> Prefer `function` for top-level function declarations and `=>` for function literals
```

Comment:
Is it worth mentioning that if you have a legitimate case for diverging from this suggestion because both have different scoping (so they are not really equal alternatives) it is ok to do so?